public/arethusa/arethusa.packages.min.js

```diff
@@ -388,11 +387,11 @@ var Z=s();"function"==typeof define&&"object"==typeof define.amd&&define.amd?(H.
 //# sourceMappingURL=toaster.min.map
-/*! angular-highlightjs
-version: 0.3.3
-build date: 2015-01-11
-author: Robin Fan
-https://github.com/pc035860/angular-highlightjs.git */
+/*! angular-highlightjs
+version: 0.3.3

```

Comment:
This file is the one that has the modified Angular code necessary to work within the React app. It needs to be reverted to the version in `master`.